tests/v0x01/test_common/test_header.py

```diff
@@ -34,11 +34,11 @@ def test_pack(self):
     def test_unpack(self):
         """[Common/Header] - unpacking Hello."""
         filename = os.path.join(os.path.dirname(os.path.realpath('__file__')),
-                                'raw/v0x01/ofpt_hello.dat')
+                                'raw/v0x01/ofpt_header.dat')
```

Comment:
@macartur, I think that there is not `ofpt_header` type for a `ofpt_header.dat` file. We were using the `ofpt_hello.dat` because the `Hello` Message only have the `Header` attribute.